llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1438,6 +1438,20 @@ def TunePreferWInst
     : SubtargetFeature<"prefer-w-inst", "PreferWInst", "true",
                        "Prefer instructions with W suffix">;
 
+let SetMaxValue = false in {
```

Comment:
Can we put this in TuneInfo instead? Seems wasteful to allocate 3 subtarget bits for something that already has command line options in the scheduler. We just need to be able to map CPU to a value just like cache size or prefetch distance right?

Reply:
Good idea! I forgot about TuneInfo. Thanks!